static/templates/edit_content_button.hbs

```diff
@@ -1,5 +1,7 @@
-{{#if is_editable}}
-<i class="fa fa-pencil edit_content_button" role="button" tabindex="0" aria-label="{{t 'Edit message' }} (e)" data-tippy-content="{{#tr}}Edit message{{/tr}} (e)"></i>
+{{#if is_content_editable}}
+<i class="fa fa-pencil edit_content_button edit_message_button" role="button" tabindex="0" aria-label="{{t 'Edit message' }} (e)" data-tippy-content="{{#tr}}Edit message{{/tr}} (e)"></i>
+{{else if can_move_message}}
+<i class="fa fa-arrows move_message_button edit_message_button" role="button" tabindex="0" aria-label="{{t 'Move message' }} (e)" data-tippy-content="{{#tr}}Move message{{/tr}}"></i>
 {{else}}
-<i class="fa fa-file-code-o edit_content_button" role="button" tabindex="0" aria-label="{{t 'View message source' }} (e)" data-tippy-content="{{#tr}}View message source{{/tr}} (e)" data-message-id="{{msg_id}}"></i>
+<i class="fa fa-file-code-o view_source_button edit_message_button" role="button" tabindex="0" aria-label="{{t 'View message source' }} (e)" data-tippy-content="{{#tr}}View message source{{/tr}} (e)" data-message-id="{{msg_id}}"></i>
```

Comment:
We want to use a common class here for handling the tippy tooltips code, so I have just use `edit_message_button` for view source icon as well, since otherwise something like `edit_or_view_source_message...` might be too long. Can change it if we want to avoid any unnecessary confusion.

Reply:
Yeah, I think that's probably fine.